src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java

```diff
@@ -42,21 +42,23 @@ public void staffCanBlockUser() {
   }
 
   @Test(groups = {"usersAndRights002"}, dependsOnMethods = {"staffCanBlockUser"})
+  @Execute(asUser = User.BLOCKED_USER)
   public void blockedUserShouldSeeMessageOnArticleEdit() {
     VisualEditModePageObject edit =
         new WikiBasePageObject().goToArticleDefaultContentEditPage(wikiURL,
             PageContent.ARTICLE_NAME_PREFIX + DateTime.now().getMillis());
 
-    edit.getGlobalNavigation().openAccountNavigation().logIn(User.BLOCKED_USER);
+    //edit.getGlobalNavigation().openAccountNavigation().logIn(User.BLOCKED_USER);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)